src/components/Common/SearchByMultipleFields.tsx

```diff
@@ -66,21 +67,15 @@ const SearchByMultipleFields: React.FC<SearchByMultipleFieldsProps> = ({
 }) => {
   const { t } = useTranslation();
   const [selectedOptionIndex, setSelectedOptionIndex] = useState(
-    initialOptionIndex || 0,
+    initialOptionIndex == -1 ? 0 : initialOptionIndex,
```

Comment:
-1 is a behaviour of `Array.find` and `SearchByMultipleFields`'s prop should be agnostic of how the index was derived.

```suggestion
    initialOptionIndex,
```